_docker/lib/options.rb

```diff
@@ -68,28 +68,41 @@ def self.parse(args)
         tasks[:supporting_services] += %w(elasticsearch mysql searchisko searchiskoconfigure)
       end
 
-      opts.on('--acceptance_test_docker [=PARALLEL_TEST]', String, 'runs the cucumber features against the local running docker stack.') do |parallel='false'|
-        ENV['PARALLEL_TEST'] = parallel
+      opts.on('--acceptance_test_docker', String, 'runs the cucumber features against the local running docker stack.') do
+        if ENV['DRIVER'].to_s.empty?
+          ENV['DRIVER']='firefox'
+        end
+
+        if ENV['PARALLEL_TEST'].to_s.empty?
+          ENV['PARALLEL_TEST']='true'
+        end
+
         tasks[:kill_all] = true
         tasks[:decrypt] = true
         tasks[:set_ports] = true
         tasks[:build] = true
-        tasks[:awestruct_up_service] =  %w(-d awestruct_preview_no_reload)
         tasks[:unit_tests] = unit_test_tasks
         tasks[:supporting_services] += %w(elasticsearch mysql searchisko searchiskoconfigure)
-        tasks[:acceptance_test_target_task] = ["--rm", "awestruct_acceptance", "bundle exec rake features PARALLEL_TEST=#{ENV['PARALLEL_TEST']}"]
+        tasks[:acceptance_test_target_task] = ["--rm", "awestruct_acceptance_docker_internal"]
       end
 
-      opts.on('--acceptance_test_target HOST_TO_TEST [=PARALLEL_TEST]', String, 'runs the cucumber features against the specified HOST_TO_TEST') do |h, parallel='false'|
-        if h.start_with?("http://docker")
+      opts.on('--acceptance_test_target HOST_TO_TEST', String, 'runs the cucumber features against the specified HOST_TO_TEST') do |host|
+        ENV['HOST_TO_TEST'] = host
+
+        driver = (ENV['DRIVER'].to_s.empty?) ? ENV['DRIVER']='firefox' : ENV['DRIVER']
+        profile = (ENV['PARALLEL_TEST'].to_s.empty?) ? ENV['PARALLEL_TEST']='true' : ENV['PARALLEL_TEST']
```

Comment:
they are doing slightly different. Second one is setting a variable which is not needed it other option. This one liner looks better than changing it to:
    
    if ENV['DRIVER'].to_s.empty?
      driver = ENV['DRIVER']='firefox' 
    else 
       driver = ENV['DRIVER']
    end